bool initialized_
std_msgs/Empty data